48_positional_encoding_timeseries/python/examples/bybit_demo.py

```
@@ -0,0 +1,238 @@
+    loader = BybitDataLoader()
+    symbol = "BTCUSDT"
+    interval = "60"  # 1 hour
+    limit = 500
+
+    try:
+        df = loader.fetch_klines(symbol, interval, limit)
+        print(f"Loaded {len(df)} candles for {symbol}")
+        print(f"Date range: {df['timestamp'].min()} to {df['timestamp'].max()}")
+        print(f"Price range: ${df['close'].min():.2f} - ${df['close'].max():.2f}")
+        use_real_data = True
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)